sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/spanner/SpannerWriteIT.java

```diff
@@ -149,7 +149,9 @@ public void testWrite() throws Exception {
 
   @After
   public void tearDown() throws Exception {
-    databaseAdminClient.dropDatabase(options.getInstanceId(), databaseName);
+    if (databaseAdminClient != null) {
```

Comment:
I think this can never be null. Can you add a `checkNotNull` on line 99, which is the only place it is assigned?